rotkehlchen/globaldb/upgrades/v5_v6.py

```diff
@@ -35,6 +34,15 @@ def _create_new_tables(cursor: 'DBCursor') -> None:
 
 def _transfer_unique_cache_data(cursor: 'DBCursor') -> None:
     log.debug('Enter _transfer_unique_cache_data')
+
+    cursor.execute(
+        'SELECT COUNT(*) FROM sqlite_master WHERE type="table" and name=?',
+        ('general_cache',),
+    )
+    if cursor.fetchone() is None:  # If the general_cache table doesn't exist then there is nothing to transfer  # noqa: E501
+        log.debug('Exit _transfer_unique_cache_data')
```

Comment:
Lefteris asked for this check to avoid the rest of the logic since if there is no general_cache table there is nothing to transfer. I improved the message a bit.